hw/ip/usbdev/dv/usbdev_sim_cfg.hjson

```diff
@@ -69,10 +69,6 @@
       name: usbdev_enable
       uvm_test_seq: usbdev_enable_vseq
     }
-    {
-      name: usbdev_enable
-      uvm_test_seq: usbdev_enable_vseq
-    }
```

Comment:
I don't think this is meant to be dropped :-)

Reply:
It was redundant, so I removed it.